### ISSUE TITLE
'The switch statement failed to match any cases' on 4.72

### ISSUE THREAD

**Issue by fredemmott** [hhvm/hhvm-autoload#55]
```
Warning: The switch statement failed to match any of the cases in /Users/fredemmott/code/xhp-lib/vendor/hhvm/hhvm-autoload/src/HHClientFallbackHandler.hack on line 215
```

**Comment by fredemmott** [comment]
Added an invariant_violation with the kind, we're getting 'typedef' now

**Comment by lexidor** [comment]
I can't close this issue. This problem has been resolved in 3.1.6.
